components/automate-ui/src/app/modules/infra-proxy/chef-server-details/chef-server-details.component.html

```diff
@@ -99,10 +99,10 @@
           <chef-table>
             <chef-thead>
               <chef-tr>
-                <chef-th>Name</chef-th>
-                <chef-th>Admin</chef-th>
-                <chef-th>Projects</chef-th>
-                <chef-th class="three-dot-column"></chef-th>
+                <chef-th  class="no_border_th">Name</chef-th>
+                <chef-th class="no_border_th">Admin</chef-th>
+                <chef-th class="no_border_th">Projects</chef-th>
+                <chef-th class="three-dot-column no_border_th"></chef-th>
```

Comment:
can we follow either Kebab case or snake case. Kebab case will be best because we are using same pattern in other places